node/src/accountant/payment_adjuster/miscellaneous/data_structures.rs

```diff
@@ -101,6 +101,11 @@ impl AffordableAndRequiredTxCounts {
     }
 }
 
+pub enum AccountsByFinalization {
+    Unexhausted(Vec<AdjustedAccountBeforeFinalization>),
+    Finalized(Vec<PayableAccount>),
```

Comment:
What does these enum variants? Their name doesn't appear to be mutually exclusive. For example, if you're using `Unexhausted`, then shouldn't the other be `Exhausted`? Similarly, if it's `Finalized`, then shouldn't the other variant be `Incomplete`?

Since they aren't mutually exclusive, I find them very confusing.